src/mediaplayer.cs

```diff
@@ -51,20 +51,27 @@ interface MPMediaItem : NSSecureCoding {
 
 #if XAMCORE_2_0
 	}
-
-	[NoMac]
-	[NoTV]
+#if MONOMAC || TVOS
+	[Static]
```

Comment:
However, we need something here to dig into the string constants like MPMediaItemPropertyTitle. [Static] was a trick I saw in AppKit already, but if there is a better way I'm all ears.